modules/library/common-index-model/src/main/java/org/fao/geonet/index/converter/FormatterConfigurationImpl.java

```diff
@@ -40,6 +49,11 @@ public String getSourceSiteTitle() {
 
   @Override
   public String buildLandingPageLink(String metadataId) {
+    if (linkToCustomMetadataUrl) {
+      return String.format("%s/%s",
+          customMetadataUrl,
```

Comment:
you might need to document that the customMetadataUrl should not end with a final '/'. Also maybe a customMetadataUrlTemplate would make more sense as a variable name ?

Reply:
> Also maybe a customMetadataUrlTemplate would make more sense as a variable name ?

sorry I misread your first comment in the PR, I thought that we could provide a url with a `{metadata-uuid}` which would be replaced, hence the "template" in the variable name, but reading the code, this is not the intent. You can forget about this part of my comment.